plugins/do-kvm.py

```diff
@@ -441,7 +441,7 @@ def set_simx_network(simx):
                 set_sriov_vfs(args, idx, mode)
 
             if have_virbr:
-                qemu_args["-netdev"].add("bridge,br=virbr0,id=net%d" %(idx))
+                qemu_args["-netdev"].add("bridge,br=virbr0,id=net%d,helper=/usr/libexec/qemu-bridge-helper" %(idx))
```

Comment:
What error do you see?

I think that we are picking qemu-bridge-helper from default path.

Reply:
Here is the error I am seeing. Btw, I noticed `mkt` uses the `helper` option for `br0`.

```sh
No br0 bridge found, using NAT networking, connected to port localhost:33497 for ssh
Input 'sriov-vfs' is not an absolute file system path, escaping is likely not going to be reversible.
qemu-system-x86_64: -netdev bridge,br=virbr0,id=net1: bridge helper failed
```

https://github.com/Mellanox/mkt/blob/eebde619015419be8c92bf75c89aac1faf5c51c0/plugins/do-kvm.py#L345

Reply:
> Here is the error I am seeing. Btw, I noticed `mkt` uses the `helper` option for `br0`.
> 
> ```shell
> No br0 bridge found, using NAT networking, connected to port localhost:33497 for ssh
> Input 'sriov-vfs' is not an absolute file system path, escaping is likely not going to be reversible.
> qemu-system-x86_64: -netdev bridge,br=virbr0,id=net1: bridge helper failed
> ```
> 
> https://github.com/Mellanox/mkt/blob/eebde619015419be8c92bf75c89aac1faf5c51c0/plugins/do-kvm.py#L345

Thanks